solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -199,9 +199,15 @@ private class ClusterStateUpdater implements Runnable, Closeable {
 
     private SolrMetricsContext clusterStateUpdaterMetricContext;
 
+    private final int minimumStateSizeForCompression;
```

Comment:
Consider "Bytes" instead of "Size" as it captures the units, and implies a size.

Reply:
`size` is better compared to bytes

`minimumBytesForCompression` vs `minimumStateSizeForCompression`

however, we could use `minStateSizeForCompression` instead of `minimumStateSizeForCompression`

Reply:
You didn't supply a reason why "size" is better than "bytes".  I gave a reason why "bytes" is better.

Other idea is "ByteLen".

Agreed on "min" vs "minimum".

Note that if we ultimately land on a unit ambiguous name (as is right now), the very least we could do is add a line of docs to specify the unit.

Reply:
I thnk solr uses `size` more often . Either is fine